content/telegraf/v1.17/concepts/aggregator_processor_plugins.md

```diff
@@ -10,37 +10,24 @@ menu:
 
 Besides the input plugins and output plugins, Telegraf includes aggregator and processor plugins, which are used to aggregate and process metrics as they pass through Telegraf.
 
-```
-┌───────────┐
-│           │
-│    CPU    │───┐
-│           │   │
-└───────────┘   │
-                │
-┌───────────┐   │                                              ┌───────────┐
-│           │   │                                              │           │
-│  Memory   │───┤                                          ┌──▶│ InfluxDB  │
-│           │   │                                          │   │           │
-└───────────┘   │    ┌─────────────┐     ┌─────────────┐   │   └───────────┘
-                │    │             │     │Aggregate    │   │
-┌───────────┐   │    │Process      │     │ - mean      │   │   ┌───────────┐
-│           │   │    │ - transform │     │ - quantiles │   │   │           │
-│   MySQL   │───┼──▶ │ - decorate  │────▶│ - min/max   │───┼──▶│   File    │
-│           │   │    │ - filter    │     │ - count     │   │   │           │
-└───────────┘   │    │             │     │             │   │   └───────────┘
-                │    └─────────────┘     └─────────────┘   │
-┌───────────┐   │                                          │   ┌───────────┐
-│           │   │                                          │   │           │
-│   SNMP    │───┤                                          └──▶│   Kafka   │
-│           │   │                                              │           │
-└───────────┘   │                                              └───────────┘
-                │
-┌───────────┐   │
-│           │   │
-│  Docker   │───┘
-│           │
-└───────────┘
-```
+{{< diagram >}}
+  graph TD
+  Process[Process<br/> - transform<br/> - decorate<br/> - filter]
+  Aggregate[Aggregate<br/> - transform<br/> - decorate<br/> - filter]
```

Comment:
Aggregate[Aggregate... - mean... - quantile... - min/max... - count]>> ?

Reply:
Curious, could we keep the left to right flow (instead of top to bottom)?

Reply:
- Because of the way the JS library renders the text, to get multi-line "node" names, you have to use a raw HTML line breaks (`<br />`).
- We definitely can, but the top-down (`TD`) layout is much more mobile friendly than the left to right (`LR`) layout.